iac-extensions/arm/src/test/java/org/sonar/iac/arm/plugin/ArmSensorTest.java

```diff
@@ -145,7 +146,9 @@ protected void verifyDebugMessages(List<String> logs) {
   @Test
   void shouldIncludeAllVisitorsInRegularContext() {
     List<TreeVisitor<InputFileContext>> visitors = sensor().visitors(context, null);
-    assertThat(visitors).hasSize(3).hasOnlyElementsOfTypes(SyntaxHighlightingVisitor.class, MetricsVisitor.class, ChecksVisitor.class);
+    assertThat(visitors)
```

Comment:
I'm confused. Why we are checking for 4 different types, but only expecting 3 visitors?

Reply:
Didn't see this change. I removed the `SyntaxHighlightingVisitor.class`